apps/microtvm/ethosu/Makefile

```
@@ -0,0 +1,110 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Makefile to build demo
+
+# Setup build environment
+build_dir := build
+STANDALONE_CRT_PATH := $(shell python3 -c "import tvm.micro; print(tvm.micro.get_standalone_crt_dir())")
+
+ARM_CPU=ARMCM55
+ETHOSU_PATH=/opt/arm/ethosu
```

```suggestion
ETHOSU_PATH ?= /opt/arm/ethosu
```
So users can specify the Ethos U path when building the demo.

---

I haven't allowed ETHOSU_PATH to be set because I don't think that makes sense in the context of the other changes I've made. But the user can set where the core driver, core platform and CMSIS code is installed:
```
ETHOSU_DRIVER_PATH ?= ${ETHOSU_PATH}/core_driver
CMSIS_PATH ?= ${ETHOSU_PATH}/cmsis
ETHOSU_PLATFORM_PATH ?= ${ETHOSU_PATH}/core_platform
```